From 431ba58daf11771195d0b494b0b0b3655a9b25cd Mon Sep 17 00:00:00 2001
From: Quentin Armitage <quentin@armitage.org.uk>
Date: Wed, 2 Aug 2017 16:18:32 +0100
Subject: [PATCH] Fix keepalived.doc(5) man page

Ryan O'Hara identified that 'vrrp_ipset' should be 'vrrp_ipsets'
in the man page.

Signed-off-by: Quentin Armitage <quentin@armitage.org.uk>
---
 doc/man/man5/keepalived.conf.5 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/man/man5/keepalived.conf.5 b/doc/man/man5/keepalived.conf.5
index 5f044af..c7b9d2e 100644
--- a/doc/man/man5/keepalived.conf.5
+++ b/doc/man/man5/keepalived.conf.5
@@ -151,7 +151,7 @@ and
  # If no names are specified, ipsets will not be used, otherwise any omitted
  # names will be constructed by adding "_if" and/or "6" to previously specified
  # names.
- vrrp_ipset [keepalived [keepalived6 [keepalived_if6]]]
+ vrrp_ipsets [keepalived [keepalived6 [keepalived_if6]]]
 
  # The following enables checking that when in unicast mode, the source
  # address of a VRRP packet is one of our unicast peers.
-- 
2.9.4

